src/navigation/sectionalTabs/__test__/sectionalTab.test.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * To run this test from the root folder, execute the following command:
+ * npx jest ./src/navigation/sectionalTabs/__test__/sectionalTab.test.js
+ */
+import { shallow } from 'enzyme';
+import React from 'react';
+import SectionalTab from '../sectionalTab';
+
+let wrapper;
+const componentProps = {
+    children: 'Example Tab',
+    classNames: '',
+    id: 'tab-exampleTab',
+    onChange: jest.fn(),
+    onClick: jest.fn(),
+    originalKey: 'exampleTab',
+    selected: false,
+};
+
+describe('<SectionalTab />', () => {
+    it('renders withouth crashing', () => {
+        wrapper = shallow(<SectionalTab {...componentProps} />);
+        expect(wrapper.length).toBe(1);
+    });
+
+    it('triggers onClick and calls onChange when both defined', () => {
+        wrapper = shallow(<SectionalTab {...componentProps} />);
+        wrapper.prop('onClick')();
+        expect(componentProps.onClick).toHaveBeenCalledTimes(1);
+        expect(componentProps.onChange).toHaveBeenCalledTimes(1);
+    });
+
+    it('has the apporpiate class when selected is true', () => {
```

Comment:
"appropriate" is misspelled here

Reply:
Thanks for the catch, I already installed this plugin so I hope not to get more further grammar errors.

![image](https://user-images.githubusercontent.com/8116485/92787583-d85ac180-f36e-11ea-9577-94f64dff796f.png)

Reply:
Fixed.  Resolving.